docs/libertyDev.md

```diff
@@ -51,3 +51,32 @@ The following are optional command line parameters supported by this task.
 ### Properties
 
 See the [Liberty server configuration](libertyExtensions.md#liberty-server-configuration) properties for common server configuration.
+
+##### System Properties for Integration Tests
+
+Integration tests can read the following system properties to obtain information about the Liberty server.
+
+| Property | Description |
+| --------  | ----------- |
+| wlp.user.dir | The user directory location that contains server definitions and shared resources. |
+| liberty.hostname | The host name of the Liberty server. |
+| liberty.http.port | The port used for client HTTP requests. |
+| liberty.https.port | The port used for client HTTP requests secured with SSL (HTTPS). |
+
+In order to properly propagate the system properties from the gradle JVM running dev mode to the JVM(s) running your integration tests, you are required to configure your `build.gradle` to set the system properties for the test JVM(s).
```

Comment:
Capitalize `Gradle`
Change `you are required to configure` to `you must configure`